src/index.ts

```diff
@@ -79,8 +79,9 @@ let answers: Answers = {};
 
 const program = new Command(packageJson.name)
     .version(packageJson.version, '-v, --version')
-    .option('-t, --type <type>', 'Solution Type: remotemonitoring, devicesimulation, devicesimulation-nohub',
-            /^(remotemonitoring|devicesimulation|devicesimulation-nohub|test)$/i,
+    .option('-t, --type <type>', 
+            'Solution Type: remotemonitoring, devicesimulation, devicesimulation-nohub, devicesimulation-vmss, devicesimulation-vmss-nohub',
+            /^(remotemonitoring|devicesimulation|devicesimulation-nohub|devicesimulation-vmss|devicesimulation-vmss-nohub|test)$/i,
```

Comment:
I still think that nohub,vmss etc. should be a skus in Device simulation. Can we discuss this and address it now?

Reply:
Good suggestion about having skus. We will consider that as a separate change. Also, for this change I've updated existing templates to use vmss instead of creating separate templates so that should minimize additional cases.